custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -189,23 +226,26 @@ def test(self, followup):
                 relationship=CASE_INDEX_EXTENSION,
                 related_type=occurrence_structure.attrs['case_type'],
             )],
+            # this prevents creating duplicate occurrence data on creation of the test cases
+            'walk_related': False,
```

Comment:
this is another notable change. without it we were copying the entire create/update blocks for the person and occurrence in with every test as well. see https://india.commcarehq.org/a/enikshay-test-2/reports/form_data/f5d4eb3a3b714957a8eb3589badd0600/ for an example of what i mean.